src/ctap/credential_id.rs

```diff
@@ -112,10 +116,16 @@ fn decrypt_cbor_credential_id(
             } else {
                 None
             };
+            let cred_blob = if let Some(blob) = cred_blob {
+                Some(extract_byte_string(blob)?)
+            } else {
+                None
+            };
```

Comment:
Here and with the above `cred_protect_policy`, a shorter way to write this is:
```
let cred_blob = cred_blob.map(extract_byte_string).transpose()?;
```

Reply:
done